gslib/gcs_json_api.py

```diff
@@ -1206,6 +1200,24 @@ def _PerformResumableDownload(self,
           retries = 0
         retries += 1
         if retries > self.num_retries:
+          ##### DEBUG
```

Comment:
Thanks for including some debug code in here, especially with multiprocessing it can get pretty hairy trying to wrestle useful info out of whichever thread has gone wild